AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -1144,9 +1136,12 @@ - (void)didMoveToWindow
   if (!visible && node.inHierarchy) {
     [node __exitHierarchy];
   }
-    
-  // Trigger updating interfaceState for cells in case ASCollectionView becomes visible or invisible
-  [_rangeController visibleNodeIndexPathsDidChangeWithScrollDirection:self.scrollDirection];
+
+  // Updating the visible node index paths only for range managed nodes. Not range managed nodes will get their
```

Comment:
comment should say only for NOT range managed collections / tables, e.g. root level ones, need this notification?  As range managed ones will get this update through the range controller itself that owns them?